ArduPlane/commands_logic.cpp

```diff
@@ -943,6 +967,69 @@ void Plane::exit_mission_callback()
     }
 }
 
+bool Plane::verify_landing_vtol_approach(const AP_Mission::Mission_Command &cmd)
+{
+    switch (vtol_approach_s.approach_stage) {
+        case LOITER_TO_ALT:
+            if (verify_loiter_to_alt()) {
+                Vector3f wind = ahrs.wind_estimate();
+                vtol_approach_s.approach_direction_deg = degrees(atan2f(-wind.y, -wind.x));
+                gcs().send_text(MAV_SEVERITY_INFO, "Selected an approach path of %f", (double)vtol_approach_s.approach_direction_deg);
```

Comment:
very minor thing - use %.1f at most. I doubt the user wants the direction to 6 decimal places.

Reply:
Done.